packages/styles/scss/components/tabs-extended/_tabs-extended.scss

```diff
@@ -109,7 +109,7 @@
           overflow: hidden;
           height: 100%;
           line-height: 1.375rem;
-          word-break: break-word;
+          hyphens: auto;
```

Comment:
Do we need this? Double checking because there's functionality to cut off words in the desktop tabs. Ends up looking like this when combined:

<img width="904" alt="Screen Shot 2023-08-21 at 2 48 06 PM" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/12532727/e4ceb641-22e1-48c3-9845-61bb4175f5e0">

Reply:
While the ellipses truncation works well, this helps to break long words in the proper location. This is easiest to test in German, as it's a well-supported language and has lots of long words to choose from, like _freundschaftsbezeugung_.

If we use that as a tab label on the [de-de version](https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/10849/iframe.html?id=components-tabs-extended--default&viewMode=story&cc=de&lc=de) of the page, we get an acceptable, automatic, hyphenation.

<img width="238" alt="Screenshot 2023-08-21 at 3 47 26 PM" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/25532785/166f3e08-00c7-4432-a194-77839badaeff">
<img width="938" alt="Screenshot 2023-08-21 at 3 47 16 PM" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/25532785/962518df-8b62-47d9-9289-6b851c1d31a1">